Dizzy/Screens/Discovery/PlacesInteractor.swift

```diff
@@ -15,12 +15,23 @@ protocol PlacesInteractorDelegate: class {
 
 protocol PlacesInteractorType {
     var delegate: PlacesInteractorDelegate? { get set }
+    var allPlaces: Observable<[PlaceInfo]> { get set }
     func getAllPlaces()
     func getProfileMedia(forPlaceId placeId: String, completion: @escaping  ([PlaceMedia]) -> Void)
     func getPlaces(ownedBy userId: String)
+    func increment(analyticsType: PlacesInteractor.AdminAnalyticsType, by count: Int, to place: PlaceInfo)
 }
 
 class PlacesInteractor: PlacesInteractorType {
+    
+    var allPlaces = Observable<[PlaceInfo]>([PlaceInfo]())
+    
+    enum AdminAnalyticsType: String {
+        case attendence = "attendenceCount"
+        case profileViews = "profileViews"
+        case reserveClicks = "reserveClicks"
+    }
+    
     weak var delegate: PlacesInteractorDelegate?
```

Comment:
now that `PlacesInteractor` is kind of singleton it does not suppose to have delegate because delegate is one to one and the places iteractor now becomes one to many, so you need to take care of that

Reply:
Done